addons/t9n/static/src/core/project_list.js

```diff
@@ -0,0 +1,87 @@
+import { Component, useState, onMounted } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { formatList } from "@web/core/l10n/utils";
+
+export class ProjectList extends Component {
+    static template = "t9n.ProjectList";
```

Comment:
If a component has no props, we explicitly declare it with an empty object:
```suggestion
    static props = {};
    static template = "t9n.ProjectList";
```
That way, it will throw an error if someone tries to pass it props anyway